packages/utilities/psammead-locales/moment/pa.js

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */
+const moment = require('moment');
+
+moment.updateLocale('pa-in', {
+  postformat: function (string) {
```

Comment:
Does `postformat: undefined` work? :)

Reply:
I just thought this might be a nicer option, but can update to your suggestions